LAYOUT_STYLE_GUIDE.md

```diff
@@ -378,6 +379,16 @@ Will result in the following output:
 
 - Next bullet.
 
+## Notes
+
+Notes can be added by wrapping the note in a `div` with class `lesson-note`. This will render the note into a box. Optionally you can add a header to the note; please wrap the header in a `h4`.
```

Comment:
Some verbiage tweaks:

```suggestion
Note boxes can be added by wrapping the content in a `div` with the class `lesson-note`. This will add styling to make the note stand out visually to users. 

A heading can be added to a note by using an `h4` element. When adding a heading, be sure to provide text that helps describe the note rather than "A note" or "Warning".
```

If a user is navigating lessons via assistive tech, a vague heading in a note may not be super helpful, especially if there are several notes with the same or similar headings.